dmapiclient/data.py

```diff
@@ -330,6 +330,11 @@ def export_users(self, framework_slug):
             "/users/export/{}".format(framework_slug)
         )
 
+    def email_address_has_valid_buyer_domain(self, email_address):
```

Comment:
`is_/has_` prefix convention works really well on property methods, eg for things like `email_address.has_valid_buyer_domain`. I think it's less useful when the owner of the method is not the subject of the "question" in the method name.

But it's just a thought, I don't really mind either way.